lib/debug/observatory_debugger.dart

```diff
@@ -287,6 +290,10 @@ class ObservatoryConnection extends DebugConnection {
       case EventKind.kIsolateExit:
         _handleIsolateDeath(ref);
         break;
+      case 'IsolateReload':
```

Comment:
Not yet - it's not part of the service protocol spec (but is supported by some VMs on tip-of-tree).